source/docs/articles/sites/database.md

```diff
@@ -9,10 +9,12 @@ Using the Workflow tool, you can overwrite the databases on your site's Dev or T
 ![Workflow Tool](/source/docs/assets/images/interface-workflow-tool.png)
 <div class="alert alert-danger"><strong>Warning:</strong>  Pushing content up to Live should almost never be done to a launched site, as it can overwrite the environment configurations. See <a href="https://pantheon.io/docs/articles/sites/code/using-the-pantheon-workflow/">Using the Pantheon Workflow</a>. </div>
 
-### MySQL Clients
+## MySQL Clients
 You can use any number of MySQL clients such as [PHPMyAdmin](/docs/articles/local/using-phpmyadmin-to-locally-administer-a-database/) or [MySQL Workbench](/docs/articles/local/using-mysql-workbench-to-access-a-database/) to  administer your site's database
 and [export configurations](/docs/articles/sites/code/using-the-pantheon-workflow/#exporting-configuration) as needed.
 
+## Cloning the Database
+Cloning relies on mysqldump, which needs a point-in-time snapshot. We recommend using the `--single-transaction` flag, which will use transactions to get a point-in-time snapshot without locking the DB. However, only the InnoDB database engine supports transactions. On MyISAM, the table needs to be locked. On small DBs this is not an issue, but could be for larger DBs. We also use the `--quick option`, which reduces the time it would take for large tables.
```

Comment:
Consider adding the link to our myisam - to - innodb article, and the stackoverflow article nick added in the issue.